lib/rouge/lexers/xojo.rb

```diff
@@ -8,7 +8,7 @@ class Xojo < RegexLexer
       desc "Xojo"
       tag 'xojo'
       aliases 'realbasic'
-      filenames '*.xojo_code', '*.xojo_window', '*.xojo_toolbar', '*.xojo_menu'
+      filenames '*.xojo_code', '*.xojo_window', '*.xojo_toolbar', '*.xojo_menu', '*.xojo_image', '*.rbbas', '*.rbfrm', '*.rbmnu', '*rbres', '*rbtbar'
```

Comment:
Are those last two supposed to have a dot after the asterisk?

Reply:
Yes. I've fixed and pushed the change.